drivers/node/ssh/ssh.go

```diff
@@ -776,6 +781,20 @@ func (s *SSH) GetBlockDrives(n node.Node, options node.SystemctlOpts) (map[strin
 	return drives, nil
 }
 
+func getExecPodNamespace() (string, error) {
```

Comment:
This method look exactly same as `GetPortworxNamespace()` in other class ? Can we avoid duplicate code or method ..

Reply:
this is a node driver and I can't invoke volume driver methods here